UnitTests/TestHelper.cs

```diff
@@ -2,8 +2,13 @@
 using GitHub;
 using GitHub.Internals;
 
-public static class TestHelper
+namespace UnitTests
```

Comment:
I'd like to remove all namespaces from the unit test projects. They don't serve a useful purpose.